exercises/verify.cpp

```diff
@@ -14,5 +14,22 @@ int main()
   // placeholder
   int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
 
+  int x;
+
+  cout<<"Enter the number to check "<<endl; 
+
+  for (int i = 0; i < 10; i++)
+  {
+    if (x=N[i])
+    {
+      cout<<"The number is on the list ";
+      break;
+    }
+    else if (i=10)
+    {
+      cout<<"The number is not on the list "<<endl; 
+    }
+  }
+
   return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```